src/components/common/HeartButton.js

```diff
@@ -0,0 +1,19 @@
+import { icFillHeartBtn, icUnfillHeartBtn } from 'assets';
+import styled from 'styled-components';
+
+function HeartButton({ isLiked, toggleHeart }) {
+  return (
+    <StyledHeartButton onClick={() => toggleHeart()}>
+      {isLiked ? <img src={icFillHeartBtn} /> : <img src={icUnfillHeartBtn} />}
```

Comment:
```suggestion
      <img src={isLiked ? icFillHeartBtn : icUnfillHeartBtn} />
```

src 안에서 삼항 연산자 쓸 수 있어~!